include/dfrn.php

```diff
@@ -424,11 +426,17 @@ public static function relocate($owner, $uid) {
 
 		$rp = q("SELECT `resource-id` , `scale`, type FROM `photo`
 				WHERE `profile` = 1 AND `uid` = %d ORDER BY scale;", $uid);
+
```

Comment:
No. Just assign the default avatars or don't include the avatars in the created XML. It would be very bad if a minor issue like a not existing avatar picture would block the transmission of the relocate message that is saying "I'm moving from server A to server B".